Realm/RealmSwift/Realm.swift

```diff
@@ -139,44 +409,102 @@ public class Realm {
         RLMAddObjectToRealm(object, rlmRealm, RLMCreationOptions.UpdateOrCreate)
     }
 
+    /**
+    Adds or updates objects in the given array to be persisted it in this Realm.
+
+    :see: addOrUpdate(object:)
+
+    :param: objects  A sequence of `Object`s to be added to this Realm.
+    */
     public func addOrUpdate<S where S: SequenceType>(objects: S) {
         for obj in objects {
             rlmRealm.addOrUpdateObject(obj as RLMObject)
         }
     }
 
+    /**
+    Deletes the given object from this Realm.
+
+    :param: object The object to be deleted.
+    */
     public func delete(object: Object) {
         RLMDeleteObjectFromRealm(object)
     }
 
+    /**
+    Deletes the given objects from this Realm.
+
+    :param: object The objects to be deleted.
+    */
     public func delete(objects: [Object]) {
```

Comment:
we should replace this (and the other one) with `delete<S: SequenceType where S.Generator.Element == Object>(objects: S)`